.github/workflows/release_nightly.yml

```diff
@@ -129,6 +136,15 @@ jobs:
           cp README.md package/README.md
           cp LICENSE.md package/LICENSE.md
 
+      - name: Package MSI
+        run: |
+          cd desktop/wix
```

Comment:
IMO we should use `desktop/packages` directory for that. There's already macos-related files, and after https://github.com/ruffle-rs/ruffle/pull/11227 there will be Linux-related metadata, so it makes sense to have Windows-installer-related things there too: `desktop/packages/windows/wix`, or `desktop/packages/wix`, or `desktop/packages/windows`.

Reply:
Sure, let's go with `packages/windows/wix`.

The macos files that are there currently aren't strictly related to distribution, it's just "the other half of the app" in mac terms, but... I'm fine with usurping the directory. :D